src/data/pokemon/evolution.h

```diff
@@ -167,8 +170,15 @@ const struct Evolution gEvolutionTable[NUM_SPECIES][EVOS_PER_MON] =
 #if P_GEN_4_POKEMON == TRUE
     [SPECIES_MURKROW]               = {{EVO_ITEM, ITEM_DUSK_STONE, SPECIES_HONCHKROW}},
     [SPECIES_MISDREAVUS]            = {{EVO_ITEM, ITEM_DUSK_STONE, SPECIES_MISMAGIUS}},
+#endif
+#if P_GEN_9_POKEMON == TRUE
+    [SPECIES_GIRAFARIG]             = {{EVO_MOVE, MOVE_TWIN_BEAM, SPECIES_FARIGIRAF}},
 #endif
     [SPECIES_PINECO]                = {{EVO_LEVEL, 31, SPECIES_FORRETRESS}},
+#if P_GEN_9_POKEMON == TRUE
+    [SPECIES_DUNSPARCE]             = {{EVO_NONE, 0, SPECIES_DUDUNSPARCE_TWO_SEGMENT},
+                                       {EVO_NONE, 0, SPECIES_DUDUNSPARCE_THREE_SEGMENT}},
```

Comment:
It evolves with Hyper Drill in SV

Reply:
I almost did that but I wasn't sure how it would work with the Dudunsparce form. I guess for now it will default to SPECIES_DUDUNSPARCE_TWO_SEGMENT

Reply:
Could you not add the method for it to evolve? Unless I'm missing something is should be doable.

Reply:
`{{EVO_MOVE, MOVE_HYPER_DRILL, SPECIES_DUDUNSPARCE_TWO_SEGMENT},`
` {EVO_MOVE, MOVE_HYPER_DRILL, SPECIES_DUDUNSPARCE_THREE_SEGMENT}},`

This is what I have now. This should work, right? Although I'm assuming it will always default to the TWO_SEGMENT form since it comes first

Reply:
It should, yes. My suggestion would be to just add a new evolution method in `GetEvolutionTargetSpecies` so it can evolve into both segments.

Reply:
> It should, yes. My suggestion would be to just add a new evolution method in `GetEvolutionTargetSpecies` so it can evolve into both segments.

Hm, it might not the way I think it could